pkg/nsx/services/ippool/builder_test.go

```diff
@@ -69,7 +69,7 @@ func TestIPPoolService_BuildIPPool(t *testing.T) {
 	vpcinfolist := []common.VPCResourceInfo{
 		{ExternalIPv4Blocks: []string{"/infra/ip-blocks/block-test"}},
 	}
-	vpcCacheIndexer := cache.NewIndexer(keyFunc, cache.Indexers{common.TagScopeVPCCRUID: indexFunc})
+	vpcCacheIndexer := cache.NewIndexer(keyFunc, cache.Indexers{common.TagScopeNetworkInfoCRUID: indexFunc})
```

Comment:
Remove this indexer?

Reply:
this CR indexer should not work, because now one networkinfo may mapping to multiple nsx-vpc

Reply:
ok